lib/WP_Auth0_Api_Client.php

```diff
@@ -1039,7 +1039,16 @@ public static function update_guardian($domain, $app_token, $factor, $enabled) {
 		return json_decode($response['body']);
 	}
 
-  protected function convertCertToPem($cert) {
+	/**
+	 * Convert a certificate to PEM format
+	 *
+	 * @see https://en.wikipedia.org/wiki/Privacy-enhanced_Electronic_Mail
+	 *
+	 * @param string $cert - certificate, like from .well-known/jwks.json
+	 *
+	 * @return string
+	 */
+  protected static function convert_cert_to_pem( $cert ) {
```

Comment:
Not sure we need to bother retiring this method and adding an alternative, it does not say public so we shouldn't have to worry about it. Also it is a very new addition so doubt anyone is using it.

Reply:
Why does WP coding guide go against PSR-2? 😢

Reply:
@cocojoe - Reverted.